tests/unit/websocket/testwebsockethandler.cpp

```diff
@@ -33,12 +33,18 @@ void MockServer::close() {
 }
 
 void MockServer::open() {
-  if (m_pWebSocketServer->listen(QHostAddress::Any, 5000)) {
+  if (m_pWebSocketServer->listen(QHostAddress::LocalHost, m_port)) {
```

Comment:
m_port is not initialized. Am I right?

Reply:
It is intentionally initialized to zero in the class declaration (`testwebsockethandler.h`). This gives us the behaviour that the port is chosen randomly from any available port when creating the class, but then persists if we open/close the websocket server.

This was needed for the `tst_reconnectionAttemptsAfterUnexpectedClose` test